configure.ac

```
@@ -6878,3 +6943,9 @@ If you want a release build with all stable optimizations active (PGO, etc),
 please run ./configure --enable-optimizations
 ])
 fi
+
+AS_VAR_IF([PY_SUPPORT_TIER], [0], [AC_MSG_WARN([
+
+Platform "$host" with compiler "$ac_cv_cc_name" is not yet supported by the
```

Last nitpick: perhaps we should remove the word 'yet', as it may make people more hopeful than needed regarding specific platform/compiler combos.

```suggestion
Platform "$host" with compiler "$ac_cv_cc_name" is not supported by the
```

---

Other than that, this PR is good to go :)